Components/Alerts/alert.css

```diff
@@ -0,0 +1,23 @@
+.alert{
+    display: flex;
+    align-items: center;
+    width: 95%;
+    max-width: 1000px;
+    height: 55px;
```

Comment:
use rem unit instead of pixels. So that by changing root font-size we can change all sizes while making responsive.